README.md

```diff
@@ -15,89 +15,75 @@ If you haven’t set up a "cloudwatch2scalyr" lambda function yet, both of these
 
-<div style="text-align:center"><img src="markdown_images/image_6.png" width="600"></div>
+<div style="text-align:center"><img src="markdown_images/image_4.png" width="600"></div>
 <br>
 
-## Define a role for your function
+## Save and test the function
 
-The last step on this page is to define a role for your function. In the next section of the page, choose "Create new role from template(s)" from the Role dropdown, give the role a name, and choose “KMS decryption permissions” from the Policy Templates dropdown:
+After you've entered all necessary information, you can save and test the function. Click on the "Save and test" button in the upper right-hand corner. On thte screen that appears, ensure that "Create new test event" is selected, then select "CloudWatch Logs" from the Event template dropdown. Give the event a name, then click "Create" at the bottom of the page.
 
-<div style="text-align:center"><img src="markdown_images/image_7.png" width="900"></div>
-<br>
-
-## Finalize the installation
-
-Now, click "Next". On the next page, click “Create Function”. You should see a screen like:
+Next, click on "Test" in the upper-right corner of the page. If the results state "Execution result: succeeded," go to the Log View in Scalyr and look for "cloudwatch-123456789123" and click on "testLogGroup" next to it.
 
-<div style="text-align:center"><img src="markdown_images/image_8.png" width="900"></div>
+You should then see two test error messages like these:
 
-At this point, you’re done!
-<br>
+<div style="text-align:center"><img src="markdown_images/image_5.png" width="600"></div>
 <br>
 
-## Try it out
+Note: If you want to change the values of the parser, log group name, or server name see the information about LOG_GROUP_OPTIONS below.
 
-Try it out by triggering some kind of action that will generate CloudWatch logs. This should cause events to be streamed to Scalyr (it may take a few seconds for them to show up):
+## Configure trigger
 
-<div style="text-align:center"><img src="markdown_images/image_9.png" width="900"></div>
-<br>
-<br>
+On the next page, you’ll configure a trigger to invoke the function. Click on the rounded dashed-line rectangle to do this and then select "CloudWatch Logs" from the dropdown that appears:
 
-As you’ll notice, the log messages that show up in Scalyr are identical to those you would see in CloudWatch. Also note the serverHost field ("cloudwatch-536….") and logfile field (“/aws/lambda/airQuality”). You can set up a custom parser in Scalyr based on the log file name (it should match your PARSER_NAME environment variable). You can also customize the serverHost field via the `SERVER_HOST` environment variable.
+<div style="text-align:center"><img src="markdown_images/image_6.png" width="600"></div>
```

Comment:
This image doesn't seem right (also it refers to 'airQualityParser' still). I also don't see image_6.png in the list of modified files for this commit - maybe you missed it?